next.config.js

```diff
@@ -0,0 +1,5 @@
+module.exports = {
```

Comment:
Looking at the [Next docs](https://nextjs.org/docs/pages/building-your-application/configuring/environment-variables), I'm pretty sure you should be able to just reference `process.env.MAPBOX_TOKEN` directly in `mapbox-map.js`

Reply:
I tried using `process.env.MAPBOX_TOKEN`, then got `undefined`. I wonder if only the server side can access directly `process.env`?

Reply:
Ahhh right I think if you prefix the env variable with `NEXT_PUBLIC_`, Next should be able to access in the browser